test-project-kotlin/greeter.provider/build.gradle.kts

```diff
@@ -2,6 +2,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 dependencies {
     implementation(project(":greeter.api"))
+    testImplementation("org.hamcrest:hamcrest:2.1-rc4")
```

Comment:
I know it's just the test project so not a big deal, but do we really need an rc version?

Reply:
Of course not. I removed it.